be/src/exprs/vectorized/bitmap_functions.cpp

```diff
@@ -366,4 +367,38 @@ ColumnPtr BitmapFunctions::bitmap_min(FunctionContext* context, const starrocks:
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+ColumnPtr BitmapFunctions::base64_to_bitmap(FunctionContext* context, const starrocks::vectorized::Columns& columns) {
+    ColumnViewer<TYPE_VARCHAR> viewer(columns[0]);
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_OBJECT> builder(size);
+    std::unique_ptr<char[]> p;
+
+    for (int row = 0; row < size; ++row) {
+        if (viewer.is_null(row)) {
+            builder.append_null();
+            continue;
+        }
+
+        auto src_value = viewer.value(row);
+        int ssize = src_value.size;
+        if (ssize == 0) {
+            builder.append_null();
+            continue;
+        }
+
+        p.reset(new char[ssize + 3]);
+
+        int decode_res = base64_decode2(src_value.data, ssize, p.get());
```

Comment:
How about using `std::string buf` and passing `buf.data()` to `base64_decode2`? In this way, we only need resize and allocate memory when the size of `row` is greater than that of the `row-1`.